src/ObjectUtils/include/ObjectUtils/PdbFile.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef OBJECT_UTILS_PDB_FILE_H_
+#define OBJECT_UTILS_PDB_FILE_H_
+
+#include <array>
+#include <filesystem>
+
+#include "OrbitBase/Result.h"
+#include "symbol.pb.h"
+
+namespace orbit_object_utils {
+
+class PdbFile {
+ public:
+  PdbFile() = default;
+  virtual ~PdbFile() = default;
+
+  [[nodiscard]] virtual ErrorMessageOr<orbit_grpc_protos::ModuleSymbols> LoadDebugSymbols() = 0;
+  [[nodiscard]] virtual const std::filesystem::path& GetFilePath() const = 0;
+
+  // GUID and Age are used to match PDB files to objects, similar to build id in the ELF case.
+  [[nodiscard]] virtual const std::array<uint8_t, 16> GetGuid() const = 0;
```

Comment:
Nit: `const` is superfluous. (Same as previous comment)

Reply:
Thanks, fixed.